packages/realm-react/README.md

```diff
@@ -250,6 +250,20 @@ const AppWrapper = () => {
   )
 }
 ```
+
+In some cases, it may be necessary to access the configured Realm from outside of the `RealmProvider`, for instance, implementing a client reset fallback.  This can be done by creating a `ref` with `useRef` and setting the `realmRef` property of `RealmProvider`.
+
+```tsx
+const AppWrapper = () => {
+  const realmRef = useRef<Realm|null>(null)
+
```

Comment:
Would it be worth showing a simple config here that uses the `realmRef`, just so it's really clear how to use this?

```suggestion
// Handle client reset errors by closing the Realm and initiating a client reset
const handleRealmSyncError: Realm.ErrorCallback = useCallback(
  (_session, error) => {
    if (realmRef.current) {
      if (error.name === 'ClientReset') {
        realmRef.current.close();
        Realm.App.Sync.initiateClientReset(realmApp, realmPath);
      }
    }
  },
  [realmPath],
);
```

Reply:
Maybe there is something even simpler we could show though...

Reply:
I originally did throw that in there, which was what inspired me to add this strategy to the `AppProvider` as well.  I would like to review how a client reset is done and see if I can simplify this.

Reply:
I think I'll leave this as is for now.  I don't want to show deprecated use cases in the documentation.